01_fib/saurabhvyas9/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// Abs returns the absolute value of x.
+func Abs(x int) int {
+	if x < 0 {
+		return -x
+	}
+	return x
+}
+
+// Returns Fibonacci and Negafibonacci series
+func fib(n int) {
+	m := Abs(n)
+	a, b, neg := 0, 1, 1
```

Comment:
`sign` instead of `neg` would have been a better choice of name imo.

Reply:
Done.